packages/astro/test/test-utils.js

```diff
@@ -22,6 +22,9 @@ polyfill(globalThis, {
 	exclude: 'window document',
 });
 
+// Disable telemetry when running tests
+process.env.ASTRO_TELEMETRY_DISABLED = true;
```

Comment:
This is where I disable the telemetry. It should affect other integration tests as they import this file too.